MODULE.bazel.lock

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
My lock file had changed -- I think since it tracks the line numbers; maybe the license addition changed the lockfile.

We should probably fail CI if this is changed during the build.
https://bazel.build/external/lockfile#lockfile-usage